### PR TITLE
Ignore exporter metrics. Non-package install instructions.

### PR DESCRIPTION
**Pull request by keithf4**
Added config option to prometheus so it stops storing the built-in metrics from postgres_exporter that we are not using.

Also updated the README's to include some instructions for non-package installs. Can work on splitting it out to different files once we start supporting other OS's besides RHEL/Centos.

### CODE REVIEW

File: prometheus/crunchy-prometheus.yml [hunk 1]
```diff
@@ -21,7 +21,12 @@ scrape_configs:
     - files:
       - /etc/prometheus/auto.d/*.yml
 
-        
+    # Do not store the metrics that come built into postgres_exporter
+    metric_relabel_configs:
+    - source_labels: [ __name__ ]
+      regex: '(pg_locks_count.*|pg_settings.*|pg_stat_activity.*|pg_stat_bgwriter.*|pg_stat_database.*)'
+          action: drop
```
Comment (sharmay) [inline prometheus/crunchy-prometheus.yml]: all our metric is prefixed ccp_ what if use that as regex and "action: keep" ?

**Comment by keithf4** [comment]
I thought about that, but then it excludes node metrics, which don't have ccp_. Also, that would exclude some of the exporter performance metrics that each have as well (we use the one to see if the exporter is up and running). I think it would be more work trying to include all the ones we want to keep vs just avoiding these few we know we don't want. Can revisit down the line if things change.

File: exporter/README.md [hunk 2]
```diff
@@ -1,17 +1,47 @@
 # Setting up exporters
 
-## Installation
+## Installation (RHEL/CENTOS 7)
 
-* Install latest node_exporter package from Crunchy Repository
-* Install latest postgres_exporter package from Crunchy Repository
-* Install latest pgmonitor-pg##-extras package for your major version of PostgreSQL
-* Install latest crunchy-pg_bloat-check package if you need to monitor database bloat
+RPMs are available to Crunchy customers through our access portal. The RPM will take care of all the steps below in the Installation section. From the Setup section onward, the instructions are relevant no matter the installation method.
```
Comment (jkatz) [inline exporter/README.md]: There are currently RPM packages available to [Crunchy Data](https://www.crunchydata.com) customers through the Crunchy Data [access portal](https://access.crunchydata.com/).  Installing the RPM will take care of all of the steps described in the installation section below.  If you install via the RPM, you can continue reading at the [Service Setup](#service-setup) section.

File: prometheus/README.md [hunk 3]
```diff
@@ -1,12 +1,24 @@
 # Setting up Prometheus
 
-## Installation
+## Installation (RHEL/CENTOS 7)
 
-* Install latest Prometheus package from Crunchy Repository
-* Install latest Alertmanager package from Crunchy Repository
-* Install latest pgmonitor-prometheus-extras package
-* Install latest pgmonitor-alertmanager-extras package
+RPMs are available to Crunchy customers through our access portal. The RPM will take care of all the steps below in the Installation section. From the Setup section onward, the instructions are relevant no matter the installation method.
```
Comment (jkatz) [inline prometheus/README.md]: There are currently RPM packages available to [Crunchy Data](https://www.crunchydata.com) customers through the Crunchy Data [access portal](https://access.crunchydata.com/).  Installing the RPM will take care of all of the steps described in the installation section below.  If you install via the RPM, you can continue reading at the [Setup](#setup) section.

**Comment by keithf4** [comment]
Updated READMEs at suggestion from @jkatz. Fixed some wording and internal section links.

**Pull request merged**